CHANGELOG

```diff
@@ -1,3 +1,17 @@
+v2.2.0
+------
+
+* Change: Allow usage of ExpressionInterface on :
+    * `ReadCommandInterface::inRows()`
+    * `ReadCommandInterface::inRow()`
+    * `Whereable::where()` on first parameter (column)
+    * `Whereable::orWhere()` on first parameter (column)
+    * `Whereable::whereNull()`
+    * `Whereable::whereNotNull()`
+
+BC Breaks
+* Change: `Bdf\Prime\Query/Expression/Raw::__toString()` is now deprecated, and its constructor value will be cast to string on constructor.
```

Comment:
Change: `Bdf\Prime\Query/Expression/Raw::__toString()` is now deprecated, and its constructor value will be cast to string